react-app/src/containers/RequestPage/RequestPage.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import logo from './logo.svg';
+import './RequestPage.css';
+import { useState } from 'react';
+import '../App.css';
+
+function RequestPage() {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [email, setEmail] = useState("");
+  const [access, setAccess] = useState("");
+
+  const handleSubmit = () => {
+    // event.preventDefault();
+    alert(`The name you entered was: ${firstName} ${lastName}\nThe email you entered was: ${email}\nThe access you chose was: ${access}`)
```

Comment:
When I tested this the popup did not display what access I chose. My name and email showed up, but access was blank.